packages/mobile/src/transactions/feed/TransactionFeed.tsx

```diff
@@ -144,6 +144,8 @@ function TransactionFeed() {
     mapStandbyTransactionToFeedTokenTransaction(tx)
   )
 
+  Logger.debug(TAG, `Standby transactions: ${JSON.stringify(standbyFeedTransactions)}`)
```

Comment:
De you think it's valuable to keep this log or was it for debugging mainly and we can delete it? I ask because the logs are already quite noisy and worry a bit about making them more so, especially because this will be printed very often I think

Reply:
I think we can remove it